github/Installation.py

```diff
@@ -52,7 +53,31 @@ def __repr__(self):
 
     @property
     def id(self):
-        return self._id
+        """
+        :type: integer
+        """
+        return self._id.value
```

Comment:
Note that this introduces a backwards incompatible change in v1.52, breaking current usage of `Installation`. Would have been nice to avoid it or at least call it out in the release notes.